impeller/aiks/experimental_canvas.cc

```diff
@@ -450,4 +467,80 @@ void ExperimentalCanvas::AddClipEntityToCurrentPass(Entity entity) {
   entity.Render(renderer_, *render_passes_.back());
 }
 
+bool ExperimentalCanvas::BlitToOnscreen() {
```

Comment:
This code is more or less identical to the code in entity_pass.cc for perfoming the same operation.